### ISSUE TITLE
Broken on latest master

### ISSUE THREAD

**Issue by quinnj** [JuliaStats/StatsBase.jl#169]
```julia
ERROR: LoadError: LoadError: UndefVarError: varzm not defined
 in include_from_node1(::String) at ./loading.jl:426 (repeats 2 times)
 in macro expansion; at ./none:2 [inlined]
 in anonymous at ./<missing>:?
 in eval(::Module, ::Any) at ./boot.jl:225
 in process_options(::Base.JLOptions) at ./client.jl:243
 in _start() at ./client.jl:322
while loading /Users/jacobquinn/.julia/v0.5/StatsBase/src/moments.jl, in expression starting on line 5
while loading /Users/jacobquinn/.julia/v0.5/StatsBase/src/StatsBase.jl, in expression starting on line 184
ERROR: LoadError: Fai
```

**Comment by malmaud** [comment]
Looks like Base doesn't define `varzm` anymore, whatever that is.

**Comment by dmbates** [comment]
There will also be problems with `Base.varzm!` in the same file.  It looks as if these were versions of `var` and `varm` that used the uncentered values, suitable for the zero-mean case.

**Comment by dmbates** [comment]
Things should work if the methods are declared as `varzm` and `varzm!` instead of `Base.varzm` and `Base.varzm!`.  These methods are used in other functions within the package.  Whether they should also be exported is something I would let @andreasnoack or @simonbyrne  decide.

**Comment by dmbates** [comment]
Methods for `Base.varzm` are also defined in the `DataArrays` package.

**Comment by andreasnoack** [comment]
I think we should simply delete `varzm`. I did a few timings and either i saw no difference or `varm` with `m=0` was actually faster. I've opened https://github.com/JuliaStats/StatsBase.jl/pull/171. For `DataArrays` see https://github.com/JuliaStats/DataArrays.jl/pull/196. Right now it fails on Travis because  it depends on `StatsBase`.

**Comment by ararslan** [comment]
Has this been fixed now that #171 has been merged?

**Comment by ararslan** [comment]
Closing as it has been fixed by subsequent releases
